doc/source/security/overview-master-key.rst

```diff
@@ -0,0 +1,396 @@
+.. _overview-master-key:
+
+==============================================================================
+Master Key Encryption Overview
+==============================================================================
+
+
+.. rubric:: System Variables
+
+.. variable:: innodb_default_encryption_key_id
```

Comment:
This is not part of Master Key encryption, but Keyring encryption.

Reply:
section removed.